tljh/traefik.py

```diff
@@ -78,6 +78,9 @@ def ensure_traefik_config(state_dir):
         os.fchmod(f.fileno(), 0o744)
         f.write(new_toml)
```

Comment:
I think we may need this one and below to have permissions `0o740` instead of `744` since there could be credentials in there now

Reply:
or `0o600` to be safe